Referentiedata.csv

```diff
@@ -989,6 +989,33 @@ INSPECTIERAPPORTSTATUS;DEF;Definitief;;;;;Onderhoud
 INSPECTIERAPPORTSTATUS;GET;Getekend;;;;;Onderhoud
 INSPECTIERAPPORTSTATUS;REV;Ter review;;;;;Onderhoud
 INSPECTIERAPPORTSTATUS;VAL;Vervallen;;;;;Onderhoud
+INSTALLATIESOORT;IAF;Inbouw afzuiginstallatie;Bij een afzuiginstallatie gaat het om een luchtafvoer met afzuiging naar buiten de woning of op basis van recirculatie met actieve koolstof- en vetfilters. Een afzuiginstallatie kan zowel een afzuig- of recirculatiekap boven de kookinstallatie zijn, als een in het aanrecht geïntegreerd afzuigsysteem.;26-07-2024;;;Vastgoed
```

Comment:
Referentiedata binnen soort staat niet alfabetisch wat wel gebruikelijk is.

Reply:
Aangepast